Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
I'm guessing I'm wondering at that point why it's worth anything else checking not just the name?